<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollisionBuild</class>
 <widget class="QDockWidget" name="CollisionBuild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>614</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>257</width>
    <height>524287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>碰撞数据编辑</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QListWidget" name="m_shapeList">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>180</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="m_capsuleEditPanel">
      <property name="title">
       <string>Parameters</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Radius:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="m_capsuleRadius">
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="m_capsuleHeight">
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="singleStep">
          <double>0.200000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <action name="m_actionCapsule3">
   <property name="text">
    <string>Capsule</string>
   </property>
   <property name="toolTip">
    <string>添加胶囊体</string>
   </property>
  </action>
  <action name="m_actionDeleteCapsule">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="m_actionSolid">
   <property name="icon">
    <iconset resource="ModelEditor.qrc">
     <normaloff>:/icon/Icon/solid.bmp</normaloff>:/icon/Icon/solid.bmp</iconset>
   </property>
   <property name="text">
    <string>Solid</string>
   </property>
  </action>
  <action name="m_actionWireFrame">
   <property name="icon">
    <iconset resource="ModelEditor.qrc">
     <normaloff>:/icon/Icon/wire.bmp</normaloff>:/icon/Icon/wire.bmp</iconset>
   </property>
   <property name="text">
    <string>WireFrame</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ModelEditor.qrc"/>
 </resources>
 <connections/>
</ui>
